### DISCUSSION TITLE
Population Prediction

### DISCUSSION THREAD

**Discussion by chjch** [SERVIR-WA/GALUP#187] [Ideas]
Here, I want to introduce a couple of sources of population data and one approach to do population prediction.

### Data

The datasets below are all in raster format.

- HDX: https://data.humdata.org/dataset/highresolutionpopulationdensitymaps-gha
- LandScan: https://landscan.ornl.gov/landscan-datasets
- NASA: https://sedac.ciesin.columbia.edu/data/collection/gpw-v4

> ⚠️  **A Caveat**.
> These datasets were derived using various data sources and algorithms. That said, you may experience dramatic differences from one dataset to another. It's up to you to choose whichever dataset you believe is closest to the "TRUTH" for your specific project.

These datasets are for the entire planet, except for the first one which is Ghana specific. So you need to do some raster clipping to get the data for the area you want after you downloaded the entire dataset.

### Prediction

In the THLD district example, I did the prediction using the function, "[Forecast Sheet](https://support.microsoft.com/en-us/office/create-a-forecast-in-excel-for-windows-22c500da-6da7-45e5-bfdc-60a7062329fd?ns)", built-in Excel. The function is under the **Data** Tab and inside the **Forecast** group.<br>
![729d4098-b4cb-4b0c-a5e1-d046a3093db9](https://user-images.githubusercontent.com/12037254/130520665-38ebd1c2-45f3-4d22-81f9-4a63341fbf8e.png)

This function employs a so-called Exponential Smoothing algorithm. You can learn the details about the algorithm at https://otexts.com/fpp3/ses.html. 

Changjie

**Comment by Sheila6868** [comment]
Dear Changjie, 
Thanks for the information.
It is very much appreciated.
Regards 
Sheila

**Reply by Achidago** [reply]
Thanks for sharing
